ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -304,7 +304,12 @@ fi
 case "${CLUSTER_TYPE}" in
 aws|aws-arm64|aws-usgov) export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred;;
 azure4|azuremag|azure-arm64) export AZURE_AUTH_LOCATION=${CLUSTER_PROFILE_DIR}/osServicePrincipal.json;;
-azurestack) export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json;;
+azurestack)
+    export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json
+    if [[ -f "${CLUSTER_PROFILE_DIR}/ca.pem" ]]; then
+        export SSL_CERT_FILE="${CLUSTER_PROFILE_DIR}/ca.pem"
+    fi
+    ;;
```

Comment:
Nit: maybe we can move this `if` outside the `switch` since on the platforms where the `ca.pem` file doesn't exist this is a no-op?

Reply:
currently, only wwt ash needs to export this ENV. Not sure whether or not ca.pem exist on other platforms with specific purpose? In order to be more safe, I add `if` blocker under azurestack.

Reply:
Ok